exonum-java-binding-common/src/main/java/com/exonum/binding/common/serialization/ZonedDateTimeSerializer.java

```diff
@@ -17,22 +17,43 @@
 
 package com.exonum.binding.common.serialization;
 
-import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+import static com.exonum.binding.common.serialization.SerializationUtils.checkLength;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.time.Instant;
+import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
 
 enum ZonedDateTimeSerializer implements Serializer<ZonedDateTime> {
   INSTANCE;
 
+  private static final int BYTE_BUFFER_CAPACITY = Long.BYTES + Integer.BYTES;
```

Comment:
It is used as a byte buffer capacity, but it seems to rather be "serialized date-time size".